open-sphere-base/mantle/src/main/java/io/opensphere/mantle/icon/impl/gui/IconRotationPane.java

```diff
@@ -11,9 +12,8 @@
 import javafx.scene.layout.HBox;
 import javafx.scene.layout.Priority;
 
-import io.opensphere.mantle.icon.IconRecord;
-
 /** Icon rotation pane. */
+@SuppressWarnings("restriction")

```

Comment:
What is this doing ?

Reply:
API restriction warnings. You're not "allowed" to use JavaFX unless you're within a JFX project, which this is technically not.

Seeing the warnings is obnoxious (look into anything in `io.opensphere.core.util.fx` as an example).

Reply:
You can get around this by using the e(fx)clipse plugins